src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { PageNotFound } from './components/PageNotFound';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" replace />} />
+        <Route path="people" element={<PeoplePage />}>
+          <Route path=":slugId" element={<PeoplePage />} />
```

Comment:
It seems like the nested route ':slugId' is rendering the same 'PeoplePage' component as its parent route. If this is intentional, consider whether the component should handle both cases differently. Otherwise, you might want to render a different component for the ':slugId' route.